notebooks/en/llm_gateway_pii_detection.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>"...how to use an example gateway for PII detection and we built upon it by adding a custom scrubber. Make sure you run your own tests to verify that whatever scrubbing algorithm you employ actually cover your use-case."</p><p> </p><p>Nice way to end the notebook 😉</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>